src/hardware/vga_other.cpp

```diff
@@ -367,11 +377,17 @@ static void update_cga16_color(void) {
 			G = pow(G, gamma);
 			B = pow(B, gamma);
 
-			int r = static_cast<int>(255*pow( 1.5073*R -0.3725*G -0.0832*B, 1/gamma)); if (r<0) r=0; if (r>255) r=255;
-			int g = static_cast<int>(255*pow(-0.0275*R +0.9350*G +0.0670*B, 1/gamma)); if (g<0) g=0; if (g>255) g=255;
-			int b = static_cast<int>(255*pow(-0.0272*R -0.0401*G +1.1677*B, 1/gamma)); if (b<0) b=0; if (b>255) b=255;
-
-			Bit8u index = bits | ((x & 1) == 0 ? 0x30 : 0x80) | ((x & 2) == 0 ? 0x40 : 0);
+			const auto r = static_cast<uint8_t>(
```

Comment:
TODO for myself: continue review here

Reply:
Prior comments have been addressed.
We're all set to continue, next chance you get @dreamer :rocket: